v03_pipeline/lib/tasks/reference_data/update_cached_reference_dataset_queries.py

```diff
@@ -11,7 +11,7 @@
 )
 
 
-class WriteCachedReferenceDatasetQuery(luigi.Task):
+class UpdateCachedReferenceDatasetQueries(luigi.Task):
```

Comment:
I renamed this to clarify that it's a wrapper around `UpdatedCachedReferenceDatasetQuery`.